scene/src/bevy-api/interface.ts

```diff
@@ -25,7 +27,25 @@ export type KernelFetchRespose = {
   body: string
 }
 
+type PBAvatarEquippedData = {
+  wearableUrns: URN[]
+  emoteUrns: URN[]
+}
+type RGBColor = { r: number; g: number; b: number }
+export type PBAvatarBase = {
+  name: string
+  skinColor?: RGBColor
+  eyesColor?: RGBColor
+  hairColor?: RGBColor
+  bodyShapeUrn: URNWithoutTokenId
+}
+
+type SetAvatarData = {
+  base?: PBAvatarBase
+  equip?: PBAvatarEquippedData
+}
 export type BevyApiInterface = {
+  setAvatar: (avatarData: SetAvatarData) => Promise<any> // TODO set profile type
```

Comment:
```suggestion
  setAvatar: (avatarData: SetAvatarData) => Promise<number> // TODO set profile type
```

if I´m not wrong. The Bevy implementation returns the deployed version, which it is a number
cc @robtfm

Reply:
https://github.com/dclexplorer/bevy-ui-scene/commit/d0b74fa20245f12fcc2f833e257d1ef5ea3301f7